Pinta.Effects/Effects/JuliaFractalEffect.cs

```diff
@@ -147,6 +158,9 @@ public sealed class JuliaFractalData : EffectData
 		[Caption ("Zoom"), MinimumValue (0), MaximumValue (50)]
 		public int Zoom { get; set; } = 1;
 
+		[Caption ("Color Scheme")]
+		public PredefinedGradients ColorScheme { get; set; } = PredefinedGradients.Bonfire;
```

Comment:
Is it possible to have the effect data just contain a `ColorGradient`, and leave it up to `SimpleEffectDialog` to render that as a list of the pre-defined gradients for now?

Reply:
It is possible, and that was the original idea, but I did it this way because it doesn't require changes in the public API.

As you've mentioned before, there are core libraries that should be sufficient for the user of the library to write new effects, and then there is `Pinta.Effects`. I think that if everything is contained in `Pinta.Effects` we can play fast and loose with the effects until we get the API just right (thus making it easier to write new effects more quickly, without worrying about the API until we get it right).

In this case, the core API is sufficient because `SimpleEffectDialog` can handle `enum` types.

But this is, of course, an opinion. What is your view, given the above?

Reply:
OK, yeah I think keeping everything in Pinta.Effects is a reasonable approach for now, then